### DISCUSSION TITLE
YOKIS gamme -UP _ Zigbee 3.0 Integration ?

### DISCUSSION THREAD

**Discussion by SebastienMrc** [Koenkk/zigbee2mqtt#24484] [Devices]
Hello, is it possible to integrate the modules of the "Yokis" brand -UP range into Zigbee2 Mqtt?
I am thinking in particular of the module: MTR1300EB-UP 

These modules work in Zigbee 3.0 and are integrated with ZHA (in Home Assistant) but I can't find any information about their compatibility with Zigbee2Mqtt.

Here is the link of the product in question: https://csa-iot.org/csa_product/mtr1300eb-up/

**Comment by hp040609** [comment]
Also very interested in the answer !

**Comment by Iroamstaff** [comment]
Il faudrait commencer par appairer le module a z2m et après suivre les indications du site z2m .
https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html
Et d'ailleurs la sans aucune indication personne ne peut vous aider

**Comment by Iroamstaff** [comment]
D'ailleurs tu n'est pas au bon endroit pour demander la prise en charge d'un nouveau appareil. Il faut aller dans problème.
https://github.com/Koenkk/zigbee2mqtt/issues

**Comment by SebastienMrc** [comment]
Thank you for your answer, but I don’t have this module yet. 
The electrician who makes my installation offers me this module but before accepting his quote I wanted to make sure that I could use it with Z2M. I know this module is compatible with ZHA, but I can’t find any compatibility info for Z2M.
And since I do not have the module in my possession, I can not test it directly.

**Reply by Iroamstaff** [reply]
Il n'est pas encore pris en charge mais finira par le devenir.

**Comment by LaurentChardin** [comment]
I have released the Yokis definition recently for a bunch of devices, and i am currently updating it to support MTR1300EB-UP (bornier à vis). The MTR1300E-UP is already supported.

I expect this module to be integrated in the next release: 
https://github.com/Koenkk/zigbee-herdsman-converters/pull/8705

You can also raise some requests to me directly here : https://github.com/LaurentChardin/zigbee-herdsman-converters/issues

Best,
